app.js

```diff
@@ -0,0 +1,39 @@
+const cheerio = require('cheerio');
+const axios = require('axios');
+const fs = require('fs');
+require('dotenv').config();
+
+async function getUrl() {
+    const res = await axios.get(process.env.PARSE_URL);
```

Comment:
If anything goes wrong with the request the error shouldn't be unhandled. You should wrap this statement with `try/catch` and display an error in the console before stopping the execution of the function.